framework/langchain/document_loader.ipynb

```diff
@@ -0,0 +1,237 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# SQLAlchemy\n",
+    "\n",
+    "This notebook demonstrates how to load documents from a [CrateDB] database,\n",
+    "using the document loader `CrateDBLoader`, which is based on [SQLAlchemy].\n",
+    "\n",
+    "It loads the result of a database query with one document per row.\n",
+    "\n",
+    "[CrateDB]: https://github.com/crate/crate\n",
+    "[SQLAlchemy]: https://www.sqlalchemy.org/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "Install required packages."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "#!pip install crash 'crate[sqlalchemy]' 'langchain[openai]'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Populate database."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001B[32mCONNECT OK\r\n",
+      "\u001B[0m\u001B[32mPSQL OK, 1 row affected (0.001 sec)\r\n",
+      "\u001B[0m\u001B[32mDELETE OK, 30 rows affected (0.010 sec)\r\n",
+      "\u001B[0m\u001B[32mINSERT OK, 30 rows affected (0.011 sec)\r\n",
+      "\u001B[0m\u001B[0m\u001B[32mCONNECT OK\r\n",
+      "\u001B[0m\u001B[32mREFRESH OK, 1 row affected (0.026 sec)\r\n",
+      "\u001B[0m\u001B[0m"
+     ]
+    }
+   ],
+   "source": [
+    "!rm -f mlb_teams_2012.sql\n",
+    "!wget --quiet https://github.com/crate-workbench/langchain/raw/cratedb/docs/extras/integrations/document_loaders/example_data/mlb_teams_2012.sql\n",
```

Comment:
Due to refactorings within the LangChain repository, this location has changed.

Reply:
The new location is https://github.com/crate-workbench/langchain/raw/cratedb/docs/docs/integrations/document_loaders/example_data/mlb_teams_2012.sql.